mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/PackageMojo.java

```diff
@@ -35,6 +40,53 @@
   @Component
   protected MavenProjectHelper helper;
 
+  /**
+   * Whether a JAR file will be created for the classes in the app. Using this optional configuration parameter will make the
+   * generated classes to be archived into a jar file and the classes directory will then be excluded from the app.
+   *
+   * @parameter expression="${archiveClasses}" default-value="false"
```

Comment:
this should be annotated with @parameter outside the comment

Reply:
This parameter shouldn't be there so I deleted it